.gitignore

```diff
@@ -0,0 +1,130 @@
+# Byte-compiled / optimized / DLL files

```

Comment:
@YiminArava4508 I added this .gitignore so that the venv stuff stays out of source control. It will also keep the __pycache__ out of source control since that stuff is also machine-dependent.